### ISSUE TITLE
python 3.6 unicode error

### ISSUE THREAD

**Issue by pc10201** [kennethreitz/delegator.py#18]
I user python 3.6 

```
delegator.py", line 197, in chain
    commands = _expand_args(command)
  File "delegator.py", line 178, in _expand_args
    if isinstance(command, (str, unicode)):
NameError: name 'unicode' is not defined
```

**Comment by Benhgift** [comment]
Same on python3.5. 

It seems to only trigger on `chain` not on `run`

**Comment by hezhiming** [comment]
I have the same exception
```
In [10]: c = delegator.chain('ls -1 /usr/bin | wc ')
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-10-e879ecd74fc3> in <module>()
----> 1 c = delegator.chain('ls -1 /usr/bin | wc ')

/usr/local/lib/python3.5/dist-packages/delegator.py in chain(command)
    195 
    196 def chain(command):
--> 197     commands = _expand_args(command)
    198     data = None
    199 

/usr/local/lib/python3.5/dist-packages/delegator.py in _expand_args(command)
    176 
    177     # Prepare arguments.
--> 178     if isinstance(command, (str, unicode)):
    179         splitter = shlex.shlex(command.encode('utf-8'))
    180         splitter.whitespace = '|'

NameError: name 'unicode' is not defined
```

#  The reason
Python 3 has remove the `unicode`, and the unicode is just `str`,    So  `delegator.py` should handle the Python compatibility

Maybe use it in Python 2. 7?

:)

**Comment by nateprewitt** [comment]
Looks like this was resolved with #13.
